protocol/messenger_chats.go

```diff
@@ -104,6 +107,56 @@ func (m *Messenger) ChatsPreview() []*ChatPreview {
 				}
 			}
 
+			switch chatPreview.MuteType {
```

Comment:
This bit I am not sure, I think we need to discuss a bit the strategy on what we want to do and how we want to handle muted timing

Reply:
Implemented the goroutine